src/core/ext/filters/client_channel/client_channel.cc.save

```diff
@@ -0,0 +1,3311 @@
+/*
```

Comment:
Was this file intended to be added in this PR?  Or is this still a WIP?

Reply:
This was added by mistake.

I'm reviving #14773, so this should be ready for review (pending a clean test run @Head)